includes/fields/class-attachment-upload.php

```diff
@@ -243,6 +243,13 @@ protected function sanitize() {
 
 		if ( empty( $this->value ) ) {
 			$this->value = null;
+		} elseif ( absint( get_option( 'hp_installed_time' ) ) > strtotime( '12 June 2024' ) ) {
```

Comment:
Please check if this part is required at all (after the change in the listing model class), I assume that it will filter images based on the field name anyway. Or this part fixes showing images in the Images field and it shows all the attached listing images by default (even those added via the Add Media button)?